src/components/contentContainer/quotesCarousel.tsx

```diff
@@ -0,0 +1,210 @@
+import {
+  ScreenSize,
+  applyMediaQueryMd,
+  applyMediaQueryLg,
+  contentMargin,
+} from '../../style/dimensions';
+
+interface Props {
+  data: ContentContainerInformationFragment & QuotesCarouselInformationFragment;
+}
+
+const slideWidth: Record<ScreenSize, number> = {
+  sm: 310,
+  md: 530,
+  lg: 530,
+};
+
+const slideMarginRight: Record<ScreenSize, number> = {
+  sm: 10,
+  md: 24,
+  lg: 24,
+};
+
+const Container = styled.div<{
+  backgroundColor?: string | null;
+}>`
+  background-color: ${({ backgroundColor }) =>
+    backgroundColor ? colors.contentColors[backgroundColor] : colors.White};
+  padding: 76px 0 40px;
+
+  ${applyMediaQueryMd(css`
+    padding: 76px 0 102px;
+  `)}
+`;
+
+const CarouselTitle = styled(Title)`
+  padding: 0 ${contentMargin.sm} 15px;
+
+  ${applyMediaQueryMd(css`
+    padding: 0 ${contentMargin.md} 34px;
+  `)}
+
+  ${applyMediaQueryLg(css`
+    padding: 0 ${contentMargin.lg} 34px;
+  `)}
+`;
+
+const CarouselWrapper = styled.div`
+  display: flex;
+  overflow: hidden;
+  padding: 0 ${contentMargin.sm};
+
+  ${applyMediaQueryMd(css`
+    padding: 0 ${contentMargin.md};
+  `)}
+
+  ${applyMediaQueryLg(css`
+    padding: 0 ${contentMargin.lg};
+  `)}
+`;
+
+const CarouselSlide = styled.div<{
+  backgroundColor?: string | null;
+}>`
+  background-color: ${({ backgroundColor }) =>
+    backgroundColor ? colors.contentColors[backgroundColor] : colors.White};
+  flex: 0 0 auto;
+  transition: all 0.5s ease;
+  height: fit-content;
+  width: ${slideWidth.sm}px;
+
+  :not(:last-child) {
+    margin-right: ${slideMarginRight.sm}px;
+  }
+
+  ${applyMediaQueryMd(css`
+    width: ${slideWidth.md}px;
+
+    :not(:last-child) {
+      margin-right: ${slideMarginRight.md}px;
+    }
+  `)}
+
+  ${applyMediaQueryLg(css`
+    width: ${slideWidth.lg}px;
+
+    :not(:last-child) {
+      margin-right: ${slideMarginRight.lg}px;
+    }
+  `)}
+`;
+
+const SlideInner = styled.div`
+  margin: 20px;
+
+  ${applyMediaQueryMd(css`
+    margin: 30px;
+  `)}
+`;
+
+const SlidesContainer = styled.div<{ currentSlide: number }>`
+  display: flex;
+  ${({ currentSlide }) =>
+    currentSlide &&
+    css`
+      transform: translateX(
+        calc(-${currentSlide * (slideWidth.sm + slideMarginRight.sm)}px)
+      );
+    `};
+  position: relative;
+  transition: all 0.5s ease;
+
+  ${applyMediaQueryMd(css<{ currentSlide: number }>`
+    ${({ currentSlide }) =>
+      currentSlide &&
+      css`
+        transform: translateX(
+          calc(-${currentSlide * (slideWidth.md + slideMarginRight.md)}px)
+        );
+      `};
+  `)}
+
+  ${applyMediaQueryLg(css<{ currentSlide: number }>`
+    ${({ currentSlide }) =>
+      currentSlide &&
+      css`
+        transform: translateX(
+          calc(-${currentSlide * (slideWidth.lg + slideMarginRight.lg)}px)
+        );
+      `};
+  `)}
+`;
+
+const AuthorImage = styled(Image)`
+  height: 64px;
+  margin-right: 13px;
+  min-width: 64px;
+  width: 64px;
```

Comment:
Maybe it would be nice to also define `background-size: cover` and `border-radius: 50% 50%` here to give them to possibility to upload non-squared images as well.

Not sure if that would work with the `gatsby-image` though.

Reply:
@mtauer  there would be `gatsby-background-image` what we use for the pageTitle/hero.

I decided against this here, because contentful has the option to already crop images into circles and i think the people who upload the images hopefully know best what part of the image to display and can select it there.